docs/FAQ

```diff
@@ -1318,7 +1319,7 @@ FAQ
   you want to change name resolver function you must rebuild libcurl and tell
   it to use a different function.
 
-  - The non-IPv6 resolver that can use one out of four host name resolve calls
+  - The non-IPv6 resolver that can use one out of four host names resolves calls
```

Comment:
I think they are "host name resolves", not "host names resolves". It resolves a "host name" and doing that several times makes "host name resolves". Right?